misk/src/main/kotlin/misk/web/interceptors/TracingInterceptor.kt

```diff
@@ -29,17 +29,8 @@ internal class TracingInterceptor internal constructor(private val tracer: Trace
 
   override fun intercept(chain: Chain): Any? {
     var result: Any? = null
-    var scope: Scope? = null
 
-    try {
-      scope = tracer.buildSpan("${chain.action}").startActive(true)
-      result = chain.proceed(chain.args)
-    } catch (exception: Exception) {
-      logger.warn("failed attempting to trace request", exception)
-      if (scope != null) Tags.ERROR.set(scope.span(), true)
```

Comment:
so what was the benefit of tagging an `ERROR`?

Reply:
Going to put this back in as a part of fixing https://github.com/square/misk/issues/125

Reply:
nice